package/network/services/hostapd/files/hostapd.sh

```diff
@@ -779,7 +779,7 @@ hostapd_set_bss_options() {
 		;;
 	esac
 
-	local auth_algs=$((($auth_mode_shared << 1) | $auth_mode_open))
+	local auth_algs="$((($auth_mode_shared << 1) | $auth_mode_open))"
```

Comment:
why? example of a fix?

Reply:
oh - oops!
that was supposed to be a separate PR.. must've cherry-picked while on the wrong branch..

Reply:
do you think i should keep it here or create another PR?